interop/xds/client/client.go

```diff
@@ -58,6 +58,7 @@ var (
 	mu               sync.Mutex
 	currentRequestID int32
 	watchers         = make(map[statsWatcherKey]*statsWatcher)
+	rpcSucceeded     bool
```

Comment:
There's no atomic boolean, but you can use int + atomic, like this: https://github.com/grpc/grpc-go/blob/master/internal/profiling/profiling.go#L49

Reply:
done